src/seedu/addressbook/data/person/Person.java

```diff
@@ -34,6 +34,17 @@ public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    String getPrintableString(Printable... printables) {
+        String result = "";
+        for (Printable p : printables) {
+            result += p.getPrintableString() + " ";
```

Comment:
For string concatenation in a loop, try to use StringJoiner for both readability and efficiency.